src/shared/containers/GetOrganizations.tsx

```diff
@@ -80,9 +81,14 @@ const GetOrganizations: FunctionComponent = () => {
 
   useEffect(() => {
     if (CLOUD && org?.id && isFlagEnabled('rudderstackReporting')) {
-      identify(meId, {email, orgID: org.id})
+      identify(meId, {
+        email,
+        orgID: org.id,
+        accountID: account.id,
+        accountName: account.name,
```

Comment:
Agreed, good call, and added in next commit. 

On the one hand, I do think we can be assured these properties will at least exist, since they are part of the default redux state. On the other hand, they're set to empty strings - and there's no point in calling `identify` until this data is retrieved from quartz. The useEffect hook will run again, and then make a single `identify` call once all properties are retrieved, which is better.